uint8 motor_id
int32 encoder_count